projects/storefrontlib/src/cms-components/navigation/navigation/config/navigation.config.ts

```diff
@@ -0,0 +1,5 @@
+import { NavigationUiConfig } from './navigation-ui-config';
+
+export const defaultCategoryNavigationConfig: NavigationUiConfig = {
```

Comment:
I think our filename should be a reflection of class/interface/variable name, so please set
```suggestion
export const defaultNavigationConfig: NavigationUiConfig = {
```
or change filename to `default-category-navigation.config.ts`
or even `defaultCategoryNavigationUIConfig` + `default-category-navigation-ui.config.ts`